builds/checkin/api_proxy.yaml

```diff
@@ -0,0 +1,80 @@
+trigger: none
```

Comment:
so this yaml is required to update in Git repo to be triggered for checkin, right? why create a new api_proxy YAML file?

Reply:
I am using my own script to build because cross doesn't work for me: scripts/linux/cross-platform-rust-build.sh --os ubuntu18.04 --arch arm32v7 --build-path edge-modules/api-proxy-module

Reply:
Based on offline discussion, Damon suggested to have its own file first.  Once stable, we can decide if it is good to merge with other yamls